pkg/oc/cli/logs/logs.go

```diff
@@ -261,13 +261,14 @@ func (o *LogsOptions) runLogPipeline() error {
 
 	switch {
 	case isBC:
-		buildName := buildutil.BuildNameForConfigVersion(bc.ObjectMeta.Name, int(bc.Status.LastVersion))
+		buildName := ocbuildapihelpers.BuildNameForConfigVersion(bc.ObjectMeta.Name, int(bc.Status.LastVersion))
 		build, _ = o.Client.Builds(o.Namespace).Get(buildName, metav1.GetOptions{})
 		if build == nil {
 			return fmt.Errorf("the build %s for build config %s was not found", buildName, bc.Name)
 		}
 		fallthrough
 	case isBld:
+		// TODO(juanvallejo): what to do about this? Const not available in external api, but we need to get rid of dependency on internal api
```

Comment:
Some of these will have to be sniped out to API. We need to check if they are worthy, though.